proposed/2022/cpm-dotnet-cli-add-package-support.md

```diff
@@ -144,6 +144,8 @@ NuGet restore operation generates `{projectName}.nuget.dgspec.json` file that fi
     1. Project1 will evaluate the Directory.Packages.props file in the Repository\Solution1\ directory.
     2. Project2 will evaluate the Directory.Packages.props file in the Repository\ directory.    
     Sourced from <https://devblogs.microsoft.com/nuget/introducing-central-package-management/>
-- The abscence of `Directory.Packages.Props` file for CPM onboarded projects fallback to the current behavior that is adding/updating `PackageReference` & `Version` to the project file. As per [Jeff Kluge's](https://github.com/jeffkl) comment [on a GitHub issue](https://github.com/NuGet/Home/issues/11903#issuecomment-1161996051), `At this time, the customers must use a file named Directory.Packages.props or set an MSBuild property to indicate what file you want to use. Using other files and importing it manually is not a supported scenario.`.
-- Scenarios `9`, `10` and `11` mentioned in the `Functional explanation` requires clarity about the command functionality. Currently there are 2 behaviors defined for these scenarios in this proposal.
-- `dotnet add package` by design supports only `PackageReference` style projects. Hence `Packages.Config` style projects remain unsupported.
+
+    Another potential solution would be to create a command line argument that takes in the `Directory.packages.props` file's path that the user intends to update, rather than choosing the closest props file by deafult. 
```

Comment:
```suggestion
    Another potential solution would be to create a command line argument that takes in the `Directory.Packages.props` file's path that the user intends to update, rather than choosing the closest props file by deafult. 
```